templates/github/.ci/scripts/validate_commit_message.py.j2

```diff
@@ -28,7 +28,8 @@ def __check_status(issue):
     gi = repo.get_issue(int(issue))
     if gi.pull_request:
         sys.exit(f"Error: issue #{issue} is a pull request.")
-    if gi.closed_at:
+    PR_BODY = os.environ.get("GITHUB_PULL_REQUEST_BODY", "")
+    if gi.closed_at and "backport" not in PR_BODY:
```

Comment:
Do we maybe want to search for "cherry picked from commit" in the commit message instead?

Reply:
Maybe even check, that a cherrypick does not contain a feature?

Reply:
done!
Now patchback PR fails for the right reason:
https://github.com/pulp/pulp_ansible/runs/4716629728?check_suite_focus=true